chapters/annotations.tex

```diff
@@ -8,6 +8,21 @@ \chapter{Annotations}\label{annotations}
 (For replaceable class declarations with a \lstinline[language=grammar]!constraining-clause! also refer to \cref{constraining-clause-annotations}.)
 The specification in this document defines the semantic meaning if a tool implements any of these annotations.
 
+\section{Notation for Annotation Definitions}\label{notation-for-annotation-definitions}
+
+Annotations are defined using the syntactic forms of Modelica component declarations and record definitions, with the following special semantics.
+If the annotation is described by a component declaration, the annotation is used in the form of a value modifier for the same name.
+If the annotation is described by a \lstinline!record! \emph{class} the annotation is used in the form of a modifier for a \lstinline!record! \emph{component} with the same name.
+
+A declaration equation for a component or record member specifies a default to be used when no corresponding annotation is given.
```

Comment:
I would like to try resolving the whole conflict according to the second alternative in https://github.com/modelica/ModelicaSpecification/pull/3496#pullrequestreview-1975646890:
```suggestion
A declaration equation for a component or record member specifies a default to be used when no corresponding annotation is given.
An array record member without declaration equation and with size specified by colon (\lstinline!:!) defaults to being empty.
```

If this is an acceptable solution, we should then use this where applicable as part of this PR.

Reply:
Ok then to get this past review.

Reply:
So @henrikt-ma any remaining issues?

Reply:
Not that I can see.  Sorry about the long delay.  Resolving.